### PR TITLE
Added error message and early JVM exit when using JDK 17 without labe…

### PR DESCRIPTION
**Pull request by hbs**
…lsid.slowimpl set

### CODE REVIEW

File: warp10/src/main/java/io/warp10/continuum/gts/GTSHelper.java [hunk 1]
```diff
@@ -107,7 +108,14 @@ public class GTSHelper {
   private static final boolean labelsIdSlowImpl;
 
   static {
-    labelsIdSlowImpl = "true".equals(System.getProperty(LABELSID_SLOWIMPL));
+    if ("17".equals(System.getProperty("java.specification.version")) && null == WarpConfig.getProperty(LABELSID_SLOWIMPL)) {
```
Comment (pi-r-p) [inline warp10/src/main/java/io/warp10/continuum/gts/GTSHelper.java:111]: Set labelsIdSlowImpl variable first, then test if jdk==17 and labelsIdSlowImpl == false.

Some user may set/keep labelsIdSlowImpl = false in their configuration.

**Review by pi-r-p — CHANGES_REQUESTED** [review]

File: warp10/src/main/java/io/warp10/continuum/gts/GTSHelper.java (hunk 1 above)
Comment (hbs) [reply warp10/src/main/java/io/warp10/continuum/gts/GTSHelper.java:111]: That's exactly what the code is doing, it only prints the warning if the labelsid.slowimpl key is not set AND the JDK is 17.

As there is no default value set for labelsid.slowimpl in the configuration the error WILL be displayed when using JDK17 unless the configuration key has been set explicitely.

Comment (pi-r-p) [reply warp10/src/main/java/io/warp10/continuum/gts/GTSHelper.java:111]: OK, this was intentional.

**Review by pi-r-p — APPROVED** [review]

**Review by stggn — APPROVED** [review]

File: warp10/src/test/java/io/warp10/continuum/gts/SubSerieTest.java [hunk 2]
```diff
@@ -16,13 +16,27 @@
 
 package io.warp10.continuum.gts;
 
+import java.io.StringReader;
```
Comment (stggn) [inline warp10/src/test/java/io/warp10/continuum/gts/SubSerieTest.java:19]: Please update the copyright date

**Review by stggn — CHANGES_REQUESTED** [review]

**Pull request merged**